deep_agent/aegra/graph.py

```
@@ -371,6 +371,47 @@ async def agent(runtime: ServerRuntime) -> Any:
+                model_name=model_name,
+                config_hash=cache_key,
+                assistant_id=assistant_id_val,
+                user_id=user_id_val,
+                refresh_token=refresh_token,
+                mcp_server_names=mcp_server_names,
+                orchestrator_config=orchestrator_cfg,
+            )
+
+            # Stash lifecycle metadata on the compiled graph for the
+            # Aegra worker to call register/deregister at run boundaries
+            setattr(compiled, "_lifecycle_enabled", True)
```

Ruff (0.16.0)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)